app/controllers/application_controller.rb

```diff
@@ -3,8 +3,8 @@
 class ApplicationController < ActionController::Base
   include Pundit
   protect_from_forgery
-  
-  include Authentication
+
+  # include Authentication
```

Comment:
There is some collision with Devise. When it's uncommented, current_user is nil.